HW1/P5/P5.txt

```diff
@@ -1 +1,160 @@
-# Your discussion here
+Note that helper functions are stored in the "network_commands" python module in the directory above this one.
+I wanted to place all of my network code in the same place.
+
+###### Shortest path #######
+
+The shortest path between 'Harvard_University' and 'Kevin_Bacon' is of length 3. One example is
```

Comment:
Ah crap you are correct, I forgot to do that. :( Sorry for the confusion. This would be a very simple fix in my code, I would just change the source and destination nodes. Good find!